src/app/components/ATIAnalytics/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import CanonicalATIAnalytics from './canonical';
+import AmpATIAnalytics from './amp';
+import { ATIProps } from './types';
+import buildATIUrl from './params';
+
+const ATIAnalytics = ({ data }: ATIProps) => {
+  const requestContext = useContext(RequestContext);
+  const serviceContext = useContext(ServiceContext);
+  const { isAmp } = requestContext;
+
+  const pageviewParams = buildATIUrl(
+    data,
+    requestContext,
+    serviceContext,
+  ) as string;
```

Comment:
Is the `as` necessary here, isn't the type just inferred from the `buildATIUrl` function?

Reply:
`pageviewParams` is defined as a string here. However, removing the `as string` as suggested then creates a type error on this prop being passed to the `amp` and `canonical` components. Making this change at the function level  and making `buildATIUrl` a string type causes a similar issue so I feel like the type isn't easily as inferred as I was expecting it to be. [https://github.com/bbc/simorgh/blob/bfbbfe1a9e89ce7583a01c28e2acd2c8c6966e2f/src/app/components/ATIAnalytics/types.ts#L65](URL)